ner_processing/master_mapping.py

```diff
@@ -63,231 +24,537 @@
         "description": "cell data",
         "decoder": decode4
     },
-    5: {
+    160: {
```

Comment:
What's the deal with reassigning numbers for these? Are you making them just directly match CAN ID now?